source/draw/Image.ooc

```diff
@@ -27,6 +27,11 @@ CoordinateSystem: enum {
 	YUpward = 0x02
 }
 
+TransformMethod: enum {
```

Comment:
So... do we plan to let this enum be used by `GpuImage` as well? Otherwise, it should be moved to `RasterImage`.